content/blog/v1-33-released.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Scratch Addons v1.33 is here!
+description: Badges for project creators in comments, a keyboard shortcut for pausing, and more!
+date: 2023-06-09T00:00:00Z
+author: DNin01
+---
+
+Welcome back! It's about time for another Scratch Addons update, so let's go over what we have added in version 1.33!
+
+## See the VIP in an instant
+
+Telling the creator of a project apart from other users in comments sections should be easy, especially when a user looks similar to the creator. One of our new addons is "Highlight project creator in comments", which does exactly what it says! If you're on a project page's comments section and you find a comment posted by the creator of that project, you'll see a "Project Creator" badge next to that user. This can help you quickly identify the real project creator.
+
+![Project Creator badge next to a comment by Za-Chary](/assets/img/release-highlights/creator-badge-comments.png)
+
+This also works on the discussion forums. The same addon will also add an "Original Poster" badge next to posts made by the user who started a forum topic.
+
+## A new Pause shortcut
+
+When you need to set the project aside quickly, you may not have time to go up and click a button, or you might be playing a game that involves mouse movements. We've updated the Pause button addon with a keyboard shortcut to help with that. Now, you can just press Alt+X on a project page to pause and resume the project! Pretty efficient, am I right?
+
+## Improved performance for Performance graphs
+
+If you've ever used the Debugger's Performance tab and you're on a slower device, you might have noticed that opening it slows things down, and you probably don't want measuring performance to impact the very performance you're measuring, so we've made a design change to the graphs in the Performance tab that should speed things up. Previously, the graph was animated and filled in, but now we've made it, by default, just a line, which is quicker to render.
+
+![A filled graph to the left and an outlined graph to the right](/assets/img/release-highlights/graph-fill-vs-line.png)
+
+But if you want the graph to be animated and you have a faster computer, you can re-enable graph animation in the addon's settings.
+
+And that's about everything. This is a pretty small update because we're also pretty busy working on supporting some changes Scratch is making, but we hope you enjoy the new features!
```

Comment:
I think this should be expanded on a little more. Should we also mention some of the other extension performance boosts or do you think those too technical for a blog post?

Reply:
Expand the whole blog post or just this line?

To mention a few more things without going into detail, we can say:
```suggestion
And that's about every new feature. We also made a few performance and stability improvements. This is a pretty small update because we're also pretty busy working on supporting some changes Scratch is making, but we hope you enjoy the new features!
```

Reply:
> Expand the whole blog post or just this line?

The work done for Scratch's update, for example, you could mention the changes done to the editor theme menu when the customizable block colours addon is enabled.

Reply:
> > Expand the whole blog post or just this line?
> 
> The work done for Scratch's update, for example, you could mention the changes done to the editor theme menu when the customizable block colours addon is enabled.

I think we might want to wait until more Scratchers are aware of these changes before describing everything related to these changes we did. I understand how it might make more sense to mention these changes now instead of by the next update (especially if that update doesn't receive a blog post) because that's when we made them, but I wouldn't go into too much detail until Scratch announces these changes.

Reply:
I think your suggestion above can be committed and everything else looks fine.